vendor/github.com/klauspost/compress/README.md

```diff
@@ -0,0 +1,438 @@
+# compress

```

Comment:
Are we wanting to suppress this README?  It is the result of `go mod vendor`. [The README is 6 years old](https://github.com/klauspost/compress/blame/master/README.md), so it feels like it was intentionally not added with [GODRIVER-2164](https://jira.mongodb.org/browse/GODRIVER-2164) ?